### ISSUE TITLE
afterFetch for \Phalcon\Mvc\Model\BehaviorInterface

### ISSUE THREAD

**Issue by raplos** [phalcon/cphalcon#1950]
Behavior's method 'notify' is not called when models data fetched. But we can define method afterFetch in model's class.

**Comment by sukobuto** [comment]
+1

**Comment by golovanov** [comment]
+1

**Comment by erickne** [comment]
+1 here.

**Comment by erickne** [comment]
+1 :+1: 

Crappy solution is extending the class: 

```php
...
    public function afterFetch()
    { 
        // Work here
    }
...
```

**Comment by josefguenther** [comment]
+1... Still an issue on 2.0.9....

**Comment by sergeyklay** [comment]
I will try to deal. Please ping me if I forget

**Comment by josefguenther** [comment]
I fixed the issue and created a pull request #11363. This should be merged into 2.1.x.
